sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
+To fill this form:
+
+1. If you know about ARM templates, or if you're interested to know about them, we recommend you to take some time to read through the ARM template we are using for this button [here](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json). You can read more about the concept of an ARM template in our [Intro to Microsoft Azure Resource Manager Templates (Video)](https://dev.to/azure/intro-to-microsoft-azure-resource-manager-templates-video-9cc).
+2. The subscription, resource group and location are required fields, they help specify where to create the KeyVault.
+3. On the "Base Name", you'll see `[resourceGroup().name]`. Remove the contents of this field and type any alphanumeric string. This string will be used to generate unique names for the resources that will be created.
+4. To fill the "Tenant Id" field, copy it from the one you'll be able to see once you go to the [overview page of the Azure Active Directory section of the Azure portal](https://ms.portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview).
+5. To fill the "Test Application Oid", you'll first need to create an Azure Active Directory application ([here's how to](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)). Once created, go to your application's overview page, and you'll see an "Object ID" property. This is the one that you want to copy and paste on the form.
+6. Finally, accept the terms and conditions, then click on the "Purchase" button.
+7. A badge will appear on the top right of the portal, indicating that the resource is being created. You can go to your resource group's page, then go to the "Deployments" section to see a history of all the deployments you've made. You'll see that those deployments that have used an ARM template will say in their names "Microsoft.Template". Once you click in one of them, you'll see at the left "Outputs" as one of the menu options. If your deployment has succeeded, you will see the name of the KeyVault that was just created.
+8. To use the Azure Active Directory Application to authenticate, go to your Azure Active Directory, go to App Registrations, go to the application name you created, then to "Certificates & secrets", and create (or copy) your application's secret.
+9. You can use the Tenant ID, the Application Client ID (also from your AAD Application's overview), the Application Client Secret (from the Certificates & secrets page of your AAD aplication) and the KeyVault name generated from the build to set the environment variables needed to run the integration tests of this repository (namely `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET` and `KEYVAULT_NAME`).
```

Comment:
Not for this particular PR but I think we can start thinking about centralizing these smaller explanation blurbs. I have something similar in one of my sample.env files to explain how to do the same thing.